src/App.js

```diff
@@ -1,16 +1,78 @@
-import React from 'react';
+import React from "react";
+import TodoForm from "./components/TodoComponents/TodoForm";
+import TodoList from "./components/TodoComponents/TodoList";
+const tasks = [
+  {
+    task: "Organize Garage",
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: "Bake Cookies",
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      tasks: tasks,
```

Comment:
I'd recommend changing at least one of these names, either the state property or the name of the array defined outside the App component up above. it works, but just to avoid potential confusion